lib/index.js

```diff
@@ -38,6 +42,35 @@ function initialize(config) {
   });
 }
 
+function filterPreviousUploads(uploadListPath, objects) {
+  var uploadedFileList;
```

Comment:
should have an early exit for uploadListPath
```
if (!uploadListPath) {
   return objects;
}
```

That way you don't need the `if (uploadListPath)` check making the code cleaner - "if there's no uploadListPath skip this whole method"